packages/config/tailwind-preset.js

```diff
@@ -140,6 +140,9 @@ module.exports = {
           900: "#f3f4f6",
         },
       },
+      screens: {
+        pwa: { raw: "(display-mode: standalone)" },
+      },
```

Comment:
I didnt even know this was possible :D

Reply:
saw this in another project and had to steal it

Reply:
the less global.css the better